package.json

```diff
@@ -59,6 +52,6 @@
     "hmpo-template-mixins": false,
     "i18n-future": false,
     "hof-controllers": false,
-    "./lib/middleware.js": false
+    "hof-middleware.js": false
```

Comment:
![](http://mrwgifs.com/wp-content/uploads/2013/07/I-Will-Find-You-And-I-Will-Kill-You-Liam-Neeson-In-Taken.gif)